.ci/pipelines/pr-gke.Jenkinsfile

```diff
@@ -20,25 +20,30 @@ pipeline {
     }
 
     stages {
-        stage('Validate Jenkins pipelines') {
-            when {
-                expression {
-                    notOnlyDocs()
+        stage('Run checks in parallel') {
+            failFast true
+            parallel {
+                stage('Validate Jenkins pipelines') {
```

Comment:
What is the benefit of running these validations every time? To get better feedback on potential errors? I wonder if we should make this conditional on a flag or something and not incur the cost of always running these.

Reply:
It's a best-effort validation to prevent submitting pipelines with invalid syntax. I agree that it seems a little overkill to run it every time. I think we've done it for the sake of simplicity, even it shouldn't be very hard to make this validation conditional by reusing what we do with `notOnlyDocs()`.